src/ucp/rma/flush.c

```diff
@@ -15,17 +15,43 @@
 #include "rma.inl"
 
 
-static void ucp_ep_flush_error(ucp_request_t *req, ucs_status_t status)
+static void
+ucp_ep_flush_request_change_uct_cmp(ucp_request_t *req, int diff,
+                                    ucp_lane_map_t new_started_lanes)
+{
+    ucs_assertv((req->send.state.uct_comp.count + diff) >= 0,
+                "req %p: comp %p count %d, diff %d", req,
+                &req->send.state.uct_comp, req->send.state.uct_comp.count,
+                diff);
+    ucs_assertv((req->send.flush.started_lanes & new_started_lanes) == 0,
```

Comment:
maybe
```suggestion
    ucs_assertv(!(req->send.flush.started_lanes & new_started_lanes),
```
also, apply for other places, because it both uses `!(expr)` and `(expr) == 0`. but should be aligned